### DISCUSSION TITLE
monitor mode in latest version immediately exits

### DISCUSSION THREAD

**Discussion by newenglanddroneservices** [abraunegg/onedrive#2818] [General]
Hello all:
First, this is a great repo, having access to my OneDrive account on Ubuntu is really nice. And the docs are quite through.

I have the lasted version installed, v2.5.0-1np1+1.1 and I'm running it on Ubuntu 24.04 LTS

As seen in the screen shot, I run onedrive with the --monitor parm. It prints out the config file was loaded, the states "Checking if --resync needed" and then returns back to the prompt. nothing is done.

![image](https://github.com/user-attachments/assets/0a88298e-7a1e-47ee-b3d6-cb5ca1102cc6)

Is there something else that needs to be set for this? The prior version would go into monitor mode and start uploading changes... and I checked the logging folder, and nothing is there.

Any ideas?
Thanks
Harold

Here is my config file:
```
sync_dir = "/Data/OneDrive"
skip_file = "~*|.~*|*.tmp"
upload_only = "true"
check_nosync = "true"
enable_logging = "true"
local_first = "true"
no_remote_delete = "true"
sync_root_files = "true"
sync_dir_permissions = "700"
sync_file_permissions = "600"
```

**Comment by abraunegg** [comment]
@newenglanddroneservices 

Firstly what is the exit code of the application when it is doing this behaviour? Use the command `echo $?` to determine that.

I would then start by adding `--verbose` to your command ..... that it is simply exiting somewhere is *odd*
